composer.json

```diff
@@ -21,7 +21,11 @@
         "phpunit/phpunit": "^4.8",
         "ptrofimov/xpmock": "^1.1",
         "dhii/php-cs-fixer-config": "dev-php-5.3",
-        "codeclimate/php-test-reporter": "<=0.3.2"
+        "codeclimate/php-test-reporter": "<=0.3.2",
+        "dhii/data-container-interface": "^0.1"
```

Comment:
The `DataNodeInterface` extends `ContainerInterface`. Does requiring the package as a dev dependency not break `DataNodeInterface` when dev dependencies are not installed?

Reply:
As the `suggests` section says, you can install `data-container-interface` to use `DataNodeInterface`. You don't have to use `DataNodeInterface` just because you installed this package. However, screw it, it deserves to be a real dep.